data/test/scenarios/README.md

```diff
@@ -0,0 +1,52 @@
+# Test scenarios
+
+This directory contains both the scenarios used by C++ unit tests and those which are
+WML unit tests.
+
+## C++ unit tests
+
+For the C++ unit tests, it is recommended to reuse the same scenario file as much as possible
+and just inject WML into it.
+
+Injection can be done by adding a config object containing event code and then registering that
+manually for `game_events`.
+
+## Manual tests
+
+The `manual_tests` subdirectory contains scenarios that expect to be run interactively, either by
+binding a hotkey for the main menu's "Choose Test Scenario" option, or with the command line
+argument `-t <testname>`.
+
+Many of these are closer to workbenches than tests, allowing developers to do some action that isn't
+automated, and then to find out whether the result matched the expectation.
+
+## Automated WML unit tests
+
+WML unit tests are self-contained scenario files to test a specific area of WML.
+
+The test result is a status code from the `unit_test_result` enum found in `game_launcher.hpp`, or
+in rare cases tests expect to be timed out by the test runner. They can be run individually with
+Wesnoth's `-u <testname>` command line argument, but are usually run by the `run_wml_tests script`
+based on the list in `wml_test_schedule`.
+
+They are unlikely to return the same status if run with `-t <testname>`. Running them with `-t` can
+still be helpful for debugging.
+
+### Guidelines for writing automated new tests
+
+Tests are generally implemented with the `GENERIC_UNIT_TEST` macro, with two leaders called Alice
+and Bob on separate keeps. If your test needs to move them adjacent to each other, consider using
+`COMMON_KEEP_A_B_UNIT_TEST` instead.
```

Comment:
Maybe clarify that this makes them start next to each other, to head off someone trying to use it to test movement.

Reply:
Done.